myProject/my_app/README.md

```
@@ -0,0 +1,235 @@
+flutter pub get
+
+# Run the app
+flutter run
+```
+
+<img src="./assets/readme/demoCell.gif" alt="MockupSamsung" width="300">
+
+### Backend Connection
+
+The app connects to the backend API:
+- **Android Emulator**: `http://10.0.2.2:3001`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)